Agda.cabal

```diff
@@ -769,7 +769,8 @@ library
 
   -- NOTE: If adding or removing flags, also change `.ghci-9.2.2`
   if impl(ghc >= 9.2.2)
-    ghc-options: -Werror=unicode-bidirectional-format-characters
+    ghc-options: -Werror=redundant-bang-patterns
```

Comment:
Should `redundant-bang-patterns` instead under `impl(ghc >= 9.2)`?
It was added in GHC 9.2.1: https://downloads.haskell.org/~ghc/9.2.2/docs/html/users_guide/9.2.1-notes.html#compiler

Reply:
You are right that `redundant-bang-patterns` could be under `impl(ghc >= 9.2)`, however I didn't include it under `impl(ghc >= 9.2)` because I didn't want complicate unnecessarily the `Agda.cabal` and the `.ghci-*` files.

In general, we should explicitly support *only* the last version of each GHC compiler series. For example, for the GHC 9.2.* family, we should only have the `.ghci-*` and `stack-*.yaml` files related to GHC 9.2.3.

Reply:
Well, even if we only ship (`.ghci` and) `stack.yaml` files for each last minor version, we should not use constraints involving minor versions in the `.cabal` file, unless it is strictly necessary. 
If you use a minor version needlessly in a constraint, you give the false impression that e.g. warning `redundant-bang-patterns` was introduced in 9.2.2 only, not already in 9.2.0.x.  Also, strictly speaking you would then have to bump all these constraints with each minor version, like from 9.2.2 to 9.2.3, then to 9.2.4 etc.  This is completely superfluous work.

I also think the `.ghci` files should only be numbered by _major_ versions, e.g.`.ghci-9.2` instead of `.ghci-9.2.2`.  There are no major changes expected in a minor release (that's the point of a minor release).

Reply:
> I also think the `.ghci` files should only be numbered by _major_ versions, e.g.`.ghci-9.2` instead of `.ghci-9.2.2`. There are no major changes expected in a minor release (that's the point of a minor release).

That is not (always) true. The warning  `unicode-bidirectional-format-characters` was added in GHC 9.0.2, removed from 9.2.1 and added back in 9.2.2. In fact, this was the reason I didn't include `redundant-bang-patterns` under `mpl(ghc >= 9.2)` and I created the file `.ghci-9.2.2` instead of `.ghci-9.2`.